### DISCUSSION TITLE
Embedding SvgEdit in different application

### DISCUSSION THREAD

**Discussion by rohitkatlaa** [SVG-Edit/svgedit#540] [Q&A]
I want to add svgedit to a different application which has its own save method and database but I am not sure how to modify the code. Can anyone help me.

**Comment by brettz9** [comment]
Hi,

You could possibly use the embedded editor (see `/editor/embedapi.html` for a demo) and/or use extensions (see `/editor/extensions/`). The PHP or other server extensions should be of interest not only if saving to a remote database but also to see how the `svgEditor.setCustomHandlers` API is used for changing the save (or export) behavior.

For further support questions, please ask on [StackOverflow](https://stackoverflow.com/tags/svg-edit).

Thanks!
